automatic/git.install/update.ps1

```diff
@@ -3,7 +3,19 @@ import-module au
 $domain = 'https://github.com'
 $releases = "$domain/git-for-windows/git/releases/latest"
 
-function global:au_BeforeUpdate { Get-RemoteFiles -Purge -NoSuffix }
+function global:au_BeforeUpdate {
+  $releaseAssets = Get-GitHubRelease -Owner 'git-for-windows' -Name 'git' -Tag $Latest.TagName | % assets
+
+  $Latest.URL32 = $releaseAssets | ? name -match "Git-.+-32-bit.exe" | % browser_download_url
+  $Latest.URL64 = $releaseAssets | ? name -match "Git-.+-64-bit.exe" | % browser_download_url
```

Comment:
Just to note @JPRuskin, I use `% browser_download_url` on purpose here (alias will be expanded automatically). If there are multiple files, the later call to `Get-RemoteFiles` will fail which we want it to do.